content/_index.zh.html

```diff
@@ -0,0 +1,224 @@
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+
+<div class="homepage">
+  <section class="hero">
+    <div class="grid-container">
+      <img src="/images/homepage/home hero blue_v2@2x.png" alt="Hero image">
+      <div class="text-container">
+        <h1>简化加密</h1>
+        <p>NavCoin是一种易于使用的数字货币，由其社区运营。在网络经济中，可编程货币降低了做&nbsp;生意的&nbsp;成本</p>
+        <a href="/en/wallets/" class="round-btn gradient-btn">Wallets <img src="/images/icons/rightward-arrow.svg" alt="rightward arrow" class="button-icon"></a>
+      </div>
+    </div>
+  </section>
+
+  <div class="spacer zero">
+    <a href="#welcome">
+      <img src="/images/homepage/down_arrow_special.svg" alt="">
+    </a>
+  </div>
+  <section class="welcome" id="welcome">
+    <h1>欢迎来到NavCoin</h1>
+    <div class="grid-container ninety-vw-container">
+      <img src="/images/homepage/Apple-Devices2@2x.png" alt="NavCoin Wallets" class="left-img">
+      <ul class="text-container">
+        <li class="">
+          <img src="/images/homepage/hom-fast-cheap.svg" alt="">
+          <h2>低摩擦支付</h2>
+          <p>专为快速、实惠和安全的对等支付而设计。一种旨在降低经营成本&nbsp;的&nbsp;网络。</p>
+        </li>
+        <li class="">
+          <img src="/images/homepage/hom-idea.svg" alt="">
+          <h2>2014年开始社区运营</h2>
+          <p>去中心化不只是一个流行词。它建立在网络的基础之上，创造一种未经许可的&nbsp;i创新&nbsp;文化。</p>
+        </li>
+        <li class="">
+          <img src="/images/homepage/hom-cfund.svg" alt="">
+          <h2>一种为自己买单的经济&nbsp;体系。</h2>
+          <p>NavCoin本身使用的是有史以来最先进的社区基金。第一个完全去中心化、完全自治、只有社区才能&nbsp;控制的&nbsp;基金。</p>
```

Comment:
This section sounds like it has been google translated

Reply:
fixed